templates/docker/Dockerfile

```diff
@@ -44,7 +44,7 @@ RUN apt-get update \
 ## NOTE(neoaggelos): Go version used to build components
 ## !!!IMPORTANT!!! Keep up to date with "snapcraft.yaml:parts.build-deps.build-snaps.go"
 ADD install-go.sh /
-RUN /install-go.sh 1.22 && ln -s /usr/local/go/bin/go /usr/local/bin/go
+RUN /install-go.sh && ln -s /usr/local/go/bin/go /usr/local/bin/go
```

Comment:
i'd expect this to read `./install.go 1.22.6`, so that changing the version is done over the Dockerfile, not inside the script itself

Reply:
Done